reverseString.js

```diff
@@ -0,0 +1,8 @@
+const reverseString = (string) => {
+  if (string.length === 0) {
+    return string;
+  }
+  return reverseString(string.substr(1)) + string[0];
+};
+
+export default reverseString;
```

Comment:
It's cool that you did recursion here! It's different from the usual solutions.

Reply:
Thank you so much Maya ;)